test/test.js

```diff
@@ -100,6 +100,22 @@ describe('single', () => {
     )
   })
 
+  test("createjs", (fn) => (done) => {
```

Comment:
can you also add a createXliff test?

Reply:
sure, createxliff and createxliff12 added